pylint/checkers/misc.py

```diff
@@ -174,10 +174,9 @@ def process_tokens(self, tokens):
             # emit warnings if necessary
             match = self._fixme_pattern.search("#" + comment_text.lower())
             if match:
-                note = match.group(1)
                 self.add_message(
                     "fixme",
-                    col_offset=comment.string.lower().index(note.lower()),
+                    col_offset=comment.start[1] + 1,
```

Comment:
If you notice here, `comment.start[1] + 1` is the starting column of the user's comment. I think tabulation's spaces have been automatically counted during tokenization, so number of spaces won't always be a fixed number.